modules/javafx.media/src/main/java/com/sun/media/jfxmedia/locator/Locator.java

```diff
@@ -349,7 +349,7 @@ public void init() throws URISyntaxException, IOException, FileNotFoundException
             if (firstSlash != -1 && uriString.charAt(firstSlash + 1) != '/') {
                 // Only one '/' after the ':'.
                 if (protocol.equals("file")) {
-                    // Map file:/somepath to file:///somepath
+                    // Map "file:/somepath" to "file:///somepath"
```

Comment:
I did this one by hand, adding the `"` chars so that the automated program wouldn't touch the `///` in `file:///`.